express/code/blocks/ax-columns/ax-columns.js

```diff
@@ -179,6 +179,78 @@ const decoratePrimaryCTARow = (rowNum, cellNum, cell) => {
   content.parentElement.prepend(links[0]);
 };
 
+function setupCornerOverlayAnimation(cell) {
+  const bottomLeftOverlay = cell.querySelector('.corner-overlay.bottom-left');
+  const bottomRightOverlay = cell.querySelector('.corner-overlay.bottom-right');
+  const topRightOverlay = cell.querySelector('.corner-overlay.top-right');
+  const topLeftOverlay = cell.querySelector('.corner-overlay.top-left');
+  const cursorOverlay = cell.querySelector('.corner-overlay.bottom-center');
+
+  // On mouse enter - only move bottom left to center
+  cell.addEventListener('mouseenter', () => {
+    bottomLeftOverlay.style.bottom = '50%';
+    bottomLeftOverlay.style.left = '50%';
+    bottomLeftOverlay.style.transform = 'translate(-50%, 50%)';
+  });
+
+  // On mouse leave - animate elements back to positions
+  cell.addEventListener('mouseleave', () => {
+    cell.classList.add('animating-out');
+
+    setTimeout(() => {
+      cell.classList.remove('animating-out');
+      // Reset bottom left
+      bottomLeftOverlay.style.bottom = '-30px';
+      bottomLeftOverlay.style.left = '-30px';
+      bottomLeftOverlay.style.transform = 'none';
+
+      // Reset bottom right
+      bottomRightOverlay.style.bottom = '-30px';
+      bottomRightOverlay.style.right = '-30px';
+      bottomRightOverlay.style.transform = 'none';
+
+      // Reset top right
+      topRightOverlay.style.top = '60px';
+      topRightOverlay.style.right = '-30px';
+      topRightOverlay.style.transform = 'none';
+
+      // Reset cursor
+      cursorOverlay.style.bottom = '-15px';
+      cursorOverlay.style.left = '80%';
+      cursorOverlay.style.transform = 'translateX(-50%)';
+
+      // Reset top left
+      topLeftOverlay.style.top = '-30px';
+      topLeftOverlay.style.left = '120px';
+      topLeftOverlay.style.transform = 'none';
+    }, 250);
+  });
+}
+
+function createCornerOverlays(cell) {
+  const overlays = [
+    { src: '/express/code/blocks/ax-columns/img/resize-button.png', class: 'top-left' },
+    { src: '/express/code/blocks/ax-columns/img/users_ui_mockup.png', class: 'top-right' },
+    { src: '/express/code/blocks/ax-columns/img/ai-image-edit.png', class: 'bottom-left', width: 47, height: 104 },
+    { src: '/express/code/blocks/ax-columns/img/gen-ai-panel.png', class: 'bottom-right' },
+    { src: '/express/code/blocks/ax-columns/img/cursor-small.svg', class: 'bottom-center', width: 26, height: 26 },
+  ];
+
+  overlays.forEach((overlay) => {
+    const img = createTag('img', {
+      class: `corner-overlay ${overlay.class}`,
+      src: overlay.src,
+      alt: '',
+      ...(overlay.width && { width: overlay.width }),
```

Comment:
Might suggest adding these to give priority to the main LCP image to help with load performance. 
fetchpriority: 'low'
loading: 'eager'